File: currency-rate-nodejs-service/src/bean/currency-rate-bean.js
class CurrencyRateBean {

    constructor(
        fromCurrency,
        toCurrency,
        exchangeRate,
        serverPort
    ) {

        this.fromCurrency = fromCurrency;
        this.toCurrency = toCurrency;
        this.exchangeRate = exchangeRate;
        this.serverPort = serverPort;

    }
}

module.exports = CurrencyRateBean

File: currency-exchange-service/src/main/java/lk/rc/ms/currencyexchangeservice/bean/CurrencyRateBean.java
package lk.rc.ms.currencyexchangeservice.bean;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class CurrencyRateBean {
    private String fromCurrency;
    private String toCurrency;
    private BigDecimal exchangeRate;
    private int serverPort;
}
